api/services/workflow_service.py

```diff
@@ -76,6 +78,38 @@ def get_published_workflow(self, app_model: App) -> Optional[Workflow]:
 
         return workflow
 
+    def get_all_published_workflow(self, app_model: App) -> list[Workflow]:
+        """
+        Get published workflow
+        """
+
+        if not app_model.workflow_id:
+            return None
+
+        # fetch published workflow by workflow_id
+        workflows = (
+            db.session.query(Workflow)
+            .filter(Workflow.app_id == app_model.id)
+            .order_by(desc(Workflow.version))
+            .limit(10)
+            .all()
+        )
+
+        if len(workflows) > 1:
+            workflows[1].version = "latest"
+
+        for workflow in workflows:
+            try:
+                # Try to parse the version as a datetime object
+                version_datetime = datetime.strptime(workflow.version, "%Y-%m-%d %H:%M:%S.%f")
+                # If successful, reformat it to the desired format
+                workflow.version = version_datetime.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
update a db model for API response it's not a good way, please consider doing this convert in the front end.

The same issue is on L99.